### PR TITLE
Added article & featured resources

### PR DESCRIPTION
**Pull request by SondraE**
Ladybug-on-leaf

## Link to Trello card (if applicable)
https://trello.com/c/c0hfu8GK/74-add-css-tricks-article-on-custom-property-stacks

## Description
I added Mia's new CSS Tricks article with a screenshot image.
I removed "feature: true" from Workshop resources and picked out resources to feature independent of type. I did this because there are very few workshops resources, and they already have images drawing attention to them.

## Impacted areas of website
- Resources list view
- "Using Custom Property Stacks" link resource detail view

---

## Reviewer tasks:

- [ ] Review the new resource's image and text
- [ ] Decide if you agree with my decisions about which resources to feature

### CODE REVIEW

**Comment by mirisuzanne** [comment]
[Add CSS Tricks Article on "custom property stacks"](https://trello.com/c/c0hfu8GK/74-add-css-tricks-article-on-custom-property-stacks)

**Comment by SondraE** [comment]
@mirisuzanne The new podcasts you added look good. I'd say if the entity calls itself a podcast, then let's categorize it as a podcast resource. I like the new image for CSS Tricks article and the demo sample in the body.

File: content/workshops/advanced-css.md [hunk 1]
```diff
@@ -12,10 +12,6 @@ tags:
   - CSS
   - Sass
 events:
-  - venue: Webconf.asia
-    url: https://2020.webconf.asia/
-    adr: Hong Kong
-    date: 2020-08-28
```
Comment (jgerigmeyer) [inline content/workshops/advanced-css.md:18]: @mirisuzanne Should this be updated on their website as well? https://2020.webconf.asia/workshop-css

Comment (mirisuzanne) [reply content/workshops/advanced-css.md:18]: Possibly? I'm never sure how to handle when I change the name of something, but it's the same general material. Also not sure how important it is to update a conference that I expect to have cancelled due to COVID? Open to thoughts on both.

**Pull request merged**